build/wpt_test.bzl

```diff
@@ -89,10 +89,11 @@ def generate_external_modules(files):
 
     for file in files.to_list():
         file_path = wd_relative_path(file)
-        if file.basename.endswith(".js"):
+        if file.extension == "js":
             entry = """(name = "{}", esModule = embed "{}")""".format(file.basename, file_path)
-        elif file.basename.endswith(".json"):
-            entry = """(name = "{}", json = embed "{}")""".format(file.basename, file_path)
+        elif file.extension == "json":
+            # TODO(soon): This is ugly but it's hard to manipulate paths in bazel
```

Comment:
Can you make sure TODO comments include actionable items? It's hard to understand what this comment does.